content/src/content/jcr_root/apps/core/wcm/sandbox/components/carousel/v1/carousel/clientlibs/site/js/carousel.js

```diff
@@ -208,13 +208,14 @@
                         indicators[i].addEventListener("click", function(event) {
                             navigateAndFocusIndicator(index);
                         });
-                        indicators[i].addEventListener("keydown", function(event) {
-                            onKeyDown(event);
-                        });
                     })(i);
                 }
             }
 
+            that._elements.self.addEventListener("keydown", function(event) {
+                onKeyDown(event);
```

Comment:
We might want to rename: `onKeyDown` -> `navigateWithKeyboard` to better reflect its purpose?

Reply:
Would rather keep this as a general keyDown handler and not introduce a purpose to the function name. I would go the other way and saw we should wrap the other events:
`onMouseEnter` `onMouseLeave`

Reply:
sure, one way or the other, as long as we are consistent with our namings :)